data_prep/mp_prep_other_inputs_one_off.py

```diff
@@ -617,4 +840,4 @@ def mp_prep_other_inputs(tile_id_list, process):
     # Checks whether the tile_id_list argument is valid
     tile_id_list = uu.tile_id_list_check(tile_id_list)
 
-    mp_prep_other_inputs(tile_id_list=tile_id_list, process=int(process))
+    mp_prep_other_inputs(tile_id_list=tile_id_list, process=str(process))
```

Comment:
What is this line doing here? I see it was here previously-- just not sure what it's for.

Reply:
This is the main part of the script that is actually running the processing function we defined above and passing it the command line arguments/ constants, correct?

Reply:
Changes made in the following commit: 064f1d8211bb345593bb2eca6e424b5b216a7475